### PR TITLE
feat: mappings for RubiconMarket, BathPair, and BathToken

### PR DESCRIPTION
**Pull request by platocrat**
Create mappings for the following contracts:
* `RubiconMarket.sol`
* `BathPair.sol`
* `BathToken.sol`

> Note: we don't need mapping file for `BathHouse.sol` because `BathHouse.sol` does not have its own events.

### CODE REVIEW

File: generated/templates.ts [hunk 1]
```diff
@@ -0,0 +1,35 @@
+// THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
Comment (platocrat) [inline generated/templates.ts:1]: _Generated_ mappings from subgraph manifest (`subgraph.yaml`).

**Comment by bghughes** [comment]
Hey @platocrat what's the status here? Any way I can be helpful/could you layout perhaps a quick roadmap for what's left :)

**Comment by platocrat** [comment]
Hey @Madalosso, i think you're working on the old `feat/mappings` branch that I merged a few days ago

**Comment by Madalosso** [comment]
Ah, my bad, I didnt realize it was merged

**Comment by Madalosso** [comment]
I just need to update the startingBlock value to the L2 mainnet address

**Comment by platocrat** [comment]
> I just need to update the startingBlock value to the L2 mainnet address

I have `660180` as the `startingBlock` on the `main` branch. Is that incorrect?

**Comment by Madalosso** [comment]
It isn't... To be honest, I'm not 100% sure about the block number on which the market contract was deployed (not very easy to find that answer using optimism.etherscan )
So I used the one that Benjamin used on our frontend project, We did this because the previous value wasnt indexing data that it suppose to (recent trade events on the mainnet L2)

**Comment by Madalosso** [comment]
`55000` is the block # used by benjamin on the frontend, we should use this one on the main branch

**Comment by platocrat** [comment]
@Madalosso i corrected the startingBlock to 55000 on the main branch, so i will close this draft PR

**Pull request closed without merging**